vignettes/quitte-data-analysis.Rmd

```diff
@@ -234,23 +239,23 @@ or across other dimensions
 
 ```{r sum variables}
 data %>% 
-    filter('2°C' == scenario, 
-           'RUS' == region,
-           2035 == period,
+    filter(scenario == '2°C', 
+           region == 'RUS',
+           period == 2035,
            variable %in% paste0('SE|Heat|', c('Coal', 'Gas'))) %>% 
     sum_total(variable, name = 'SE|Heat|fossil')
 ```
 
 
 # Add New Variables
 `calc_addVariable()` calculates new variables from existing ones, using generic 
-formulas. Variable names, that are not valid R identifiers -- basically 
-everything inside a `.mif`-file, needs to be escaped using backticks ("`").
+formulas. Variable names that are not valid R identifiers -- basically 
+everything inside a `.mif`-file, need to be escaped using backticks ("`").
```

Comment:
Are you sure about dropping the *s*? Sounds wrong to me but I don't feel qualified to judge.